src/components/Screen.tsx

```diff
@@ -42,27 +41,25 @@ interface ViewConfig extends View {
   };
 }
 
-export class InnerScreen extends React.Component<ScreenProps> {
-  private ref: React.ElementRef<typeof View> | null = null;
-  private closing = new Animated.Value(0);
-  private progress = new Animated.Value(0);
-  private goingForward = new Animated.Value(0);
+export const InnerScreen = React.forwardRef<View, ScreenProps>(
+  function InnerScreen(props, ref) {
+    const innerRef = React.useRef<ViewConfig | null>(null);
+    React.useImperativeHandle(ref, () => innerRef.current!, []);
```

Comment:
What is this line for?

Reply:
This code is intended to replace removed code mentioned above:
`  setNativeProps(props: ScreenProps): void {
    this.ref?.setNativeProps(props);
  }`
  However I am not sure wether it is correctly implemented